src/en/zinchanmanga/src/eu/kanade/tachiyomi/extension/en/zinchanmanga/ZinChanManga.kt

```diff
@@ -5,8 +5,8 @@ import eu.kanade.tachiyomi.source.model.SManga
 import org.jsoup.nodes.Document
 import org.jsoup.nodes.Element
 
-class ZinChanManga : Madara("ZinChanManga", "https://zinchanmanga.net", "en") {
-    override val versionId = 2
+class ZinChanManga : Madara("ZinChanManga", "https://zinchangmanga.net", "en") {
+    override val versionId = 3
```

Comment:
no need to bump versionId if they are the same site or have same urls. (bumping it will create a new id so existing library entries will no longer link with the extension)